gcloud/storage/_helpers.py

```diff
@@ -124,58 +115,44 @@ def _patch_properties(self, properties):
         :rtype: :class:`_PropertyMixin`
         :returns: The current object.
         """
-        # Pass '?projection=full' here because 'PATCH' documented not
-        # to work properly w/ 'noAcl'.
-        self._properties = self.connection.api_request(
-            method='PATCH', path=self.path, data=properties,
-            query_params={'projection': 'full'})
+        self._changes.update(properties.keys())
```

Comment:
We don't really need to pass `keys()`:  a set updates fine from the dict itself.